packages/devtools_app/lib/src/table.dart

```diff
@@ -1250,8 +1254,9 @@ class _TableRowState<T> extends State<TableRow<T>>
         );
       }
 
-      content = SizedBox(
+      content = Container(
         width: columnWidth,
+        padding: const EdgeInsets.only(right: borderPadding),
```

Comment:
having right padding for the last cell in the table seems undesirable. Let me see if I can bring back that old CL I did to try to fix the padding on the table the right way.

Reply:
> having right padding for the last cell in the table seems undesirable

In practice - as long as a cell doesn't draw a different background color - this works pretty well (and is strictly better than no padding).

> Let me see if I can bring back that old CL I did to try to fix the padding on the table the right way.

Sure - happy either way.